PhotoMap/Core/Base/Models/PhotoAnnotation.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PhotoAnnotation.swift
+//  PhotoMap
+//
+//  Created by Dzmitry Makarevich on 6/10/21.
+//
+
+import MapKit
+
+class PhotoAnnotation: NSObject, MKAnnotation {
+    private let photo: Photo
+    var id: String { photo.id }
+    var title: String? {
+        photo.description == "" ? " " : photo.description
+    }
```

Comment:
Can be simplified to:
```suggestion
    var title: String? {
        photo.description
    }
```